app/dependencies.php

```diff
@@ -109,7 +109,7 @@ function (ContainerInterface $c): Auth\DonationPublicAuthMiddleware {
              */
             $settings = $c->get('settings');
             $stripeChannel = $settings['notifier']['slack']['stripe_channel'];
-            /** @var SlackChannelChatterFactory $chatterFactory */
+
```

Comment:
Things like this are no longer needed. With https://github.com/Lctrs/psalm-psr-container-plugin Psalm automatically knows that `$chatterFactory` is an instance of `SlackChannelChatterFactory`.